src/test/java/com/anusha/testing/AppTest.java

```diff
@@ -42,4 +51,265 @@ public void shouldMultiplyTwoIntegers(){
         Assert.assertEquals(new Integer(8),result);
 
     }
+    @Test
+    public void shouldAddTwoStrings(){
+        String a = "Hello";
+        String b = "World";
+
+        String c = a+b;
+
+      //  Assert.assertEquals("HelloWorld",c);
+
+        assertThat(c, is("HelloWorld"));
```

Comment:
this is very good use of using hamcrest matchers .. please can you update other string tests to use the same

Reply:
has been applied changes to the string program with hamcrest matchers